src/api.coffee

```diff
@@ -17,9 +17,15 @@ module.exports = (application) ->
 	parsedRouter.use(bodyParser())
 
 	api.use (req, res, next) ->
+		queryKey = req.query.apikey ? ''
```

Comment:
You check for `queryKey?` below, but this line sets a default so that one will never be nullish